src/pages/ChessNews.tsx

```diff
@@ -1,103 +1,98 @@
 import { useState, useEffect } from 'react';
 import { motion } from 'framer-motion';
-import axios from 'axios';
 import Sidebar from '@/components/sidebar';
 import { FaChessKnight, FaExternalLinkAlt } from 'react-icons/fa';
 import chessbaseLogo from '@/assets/cbi.svg';
 import { useLocation } from 'react-router-dom';
-
-interface Article {
-  image: string;
-  title: string;
-  summary: string;
-  url: string;
-}
+import { useChessNewsQuery } from '@/hooks/useChessNews';
 
 export default function ChessNews() {
-  const [articles, setArticles] = useState<Article[]>([]);
-  const [loading, setLoading] = useState(true);
+  const { data: articles, isLoading } = useChessNewsQuery();
+
   const [showSidebar, setShowSidebar] = useState(false);
   const location = useLocation();
 
   useEffect(() => {
-    const fetchArticles = async () => {
-      try {
-        const response = await axios.get('https://api2.bmsamay.com/latest_articles');
-        setArticles(response.data);
-        setLoading(false);
-      } catch (error) {
-        console.error('Error fetching articles:', error);
-        setLoading(false);
-      }
-    };
-
-    fetchArticles();
-
     // Check for token in URL or localStorage
     const searchParams = new URLSearchParams(location.search);
     const token = searchParams.get('token') || localStorage.getItem('token');
     setShowSidebar(!!token);
   }, [location.search]);
 
   return (
-    <div className="flex h-screen bg-gray-900 text-white">
+    <div className='flex h-screen bg-gray-900 text-white'>
```

Comment:
@Bot-Rakshit  I guess we need to set up eslint settings to honor uniformity.

Reply:
This is prettier doing this, ig there should be a root prettier config file